app/bundles/CoreBundle/Helper/BuilderTokenHelper.php

```diff
@@ -70,8 +94,12 @@ public function getTokens(
         $valueColumn = 'id',
         CompositeExpression $expr = null
     ) {
+        if (!$this->isConfigured) {
+            throw new \Exception('You must call the "configure" method of this class first.');
```

Comment:
This is great that it will fail fast with the explanation why. Love it! I think it could be improved a little more. Like this:
```suggestion
            throw new \BadMethodCallException('You must call the "'.get_class($this).'::configure()" method first.');
```
Sometimes the exceptions are logged without any stack trace. It helps to be concrete.

And update the exception in the method comments as well, please.

Reply:
Good suggestion, I agree its better to be explicit